### ISSUE TITLE
PlaceOrder ->> `Api error: Missing parameter price`

### ISSUE THREAD

**Issue by peepopoggers** [fabianboesiger/ftx#86]
When doing:
```rs
api.request(PlaceOrder {
    market: o.pair.to_string(),
    side: if o.islong {
        ftx::rest::Side::Buy
    } else {
        ftx::rest::Side::Sell
    },
    price: None,
    r#type: ftx::rest::OrderType::Market,
    size: o.real_quantity,
    client_id: None,
    ioc: false,
    post_only: false,
    reduce_only: false,
    reject_on_price_band: false,
})
.await?
```

It gives the error: `Api error: Missing parameter price`. I am confused because it says in the API to pass null for market orders. Am I missing something? This worked a few weeks ago but maybe I changed something.

**Comment by MaxFangX** [comment]
Have you updated to the latest version? This came up in #80 and was fixed in #81

**Comment by peepopoggers** [comment]
Did not spot that fix, my bad. :+1:
